guacamole/src/main/webapp/translations/en.json

```diff
@@ -364,6 +364,7 @@
         "FIELD_HEADER_SECURITY"        : "Security mode:",
         "FIELD_HEADER_SERVER_LAYOUT"   : "Keyboard layout:",
         "FIELD_HEADER_SFTP_DIRECTORY"             : "Default upload directory:",
+        "FIELD_HEADER_SFTP_HOST_KEY"              : "SSH host key (Base64):",
```

Comment:
As the fact this is the key of the SSH host is given from the header for the section (SFTP), but the public vs. private nature of that key is not, I suggest using the same text for VNC/RDP that you use for the SSH parameter ("public host key").

Reply:
Yeah, good point.  Fixed.